ethereum/contracts/TestBasicInboundChannelProxy.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./ParachainLightClient.sol";
+import "./BasicInboundChannel.sol";
+
+contract TestBasicInboundChannelProxy is Ownable {
```

Comment:
Could you add a comment describing what the idea of this contract is? Why do we test this case?

Reply:
renamed and commented